hogben/models/samples.py

```diff
@@ -159,13 +161,6 @@ def _get_sld_profile(self):
         if isinstance(self.structure, refnx.reflect.Structure):
             z, slds = self.structure.sld_profile()
 
-        # Determine if the structure was defined in Refl1D.
```

Comment:
This logic was removed because `hogben.simulate.refl1d_experiment` no longer exists, and thus the import lead to an error. Leaving this function there anyway with a non-existing function call is probably not great, and the linter complains about that as well. Same with other references to `refl1d_experiment`.